pkg/networkpolicy/packet_test.go

```diff
@@ -0,0 +1,231 @@
+package networkpolicy
+
+import (
+	"net"
+	"testing"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+// captureChunk is the number of bytes transfered to user-space
+// in the nfqueue
+const captureChunk = 128
+
+func TestUdpFragmentIPv6(t *testing.T) {
```

Comment:
"UDP" not "Udp". Golang style doesn't force acronyms to "titlecase" (though they can be all-lowercase if needed).

(Likewise everywhere.)